app/src/conferences/models.py

```diff
@@ -341,6 +337,10 @@ def toggle_payment_accepted(self):
     def room(self):
         return self.user.room_of_user
 
+    @property
+    def transfer_title(self):
+        return f"ZOSIA {self.user.first_name} {self.user.last_name} {self.user.hash}"
```

Comment:
Shouldn't we add some dashes here? Something like `ZOSIA - John Smith - 123456af`?